### PR TITLE
Updating host specs for smart start installs based on resource needs …

### PR DESCRIPTION
**Pull request by etsauer**
…for EFK and some confusing language about the 'Root Drive' vs the '/var/' partition

@JaredBurck @sabre1041 @jkupferer

### CODE REVIEW

**Comment by etsauer** [comment]
@cpitman @mwitzenm @sabre1041 @timothyh please review sizing updates.

**Comment by bbeaudoin** [comment]
The numbers are very debatable so I won't. It's too easy for me to argue both ways before posting a comment and I can imagine the recommended numbers for these engagements to only go up.

We might be using partitions and logical volumes interchangeably, to Tim's point we should likely have a clear recommendation bias toward LVM.

**Comment by etsauer** [comment]
@timothyh @bbeaudoin  the specs that I came up with for the masters are based on the following documentation.

https://docs.openshift.com/container-platform/latest/install_config/install/prerequisites.html#production-level-hardware-requirements

Based on that, I came up with the following:

Minimum system requirements: 2 core, 8GB (it says 16GB, but i don't believe it)
+
planning for initial 4000 pods, 4 cores, 6GB ram
+
2 core, 8GB for etcd. (docs say the above sizing assumes external etcd, which we now know that we don't actually recommend).

From there I just rounded the 6GB up to 8GB for a nicer number, and got 8x24.

I'm open to making this number smaller, but I want to have a reason that we're going against documentation.

I am fine with bumping up the memory on the nodes.. i think 48 is reasonable.

**Comment by etsauer** [comment]
@bbeaudoin @timothyh @mwitzenm made updates based on your feedback. look better now?

**Comment by mwitzenm** [comment]
Looks good to me!!

**Comment by etsauer** [comment]
@sabre1041 would you mind giving me a reivew/merge here?

**Comment by sabre1041** [comment]
@etsauer looks good. Only comment is to add an additional application node to the inventory file as 3 is the recommended count.

**Comment by etsauer** [comment]
@sabre1041 done.

**Comment by etsauer** [comment]
Hold off on merging. We had another discussion about specs and I am adjusting the sizing.

**Comment by bparry02** [comment]
@etsauer with the diagram change, it no longer matches the introductory text.  `It consists of 3 Masters and 5 Nodes.` There are now 6 Nodes in the diagram. Is the intention to have 2 infra nodes or 3?

**Comment by etsauer** [comment]
@bparry02 the intent is 3 infranodes. I'll update the diagram with that.

File: playbooks/installation/installation.adoc [hunk 1]
```diff
@@ -50,28 +50,32 @@ The Smart Start architecture requires the following Servers(VMs) be provisioned:
```
Comment (JaredBurck) [inline playbooks/installation/installation.adoc]: line 20 above, second sentence, first word - replace "Two" with "Three" to reflect the number of infra nodes in the updated diagram.

line 30 above -  update '2 infrastructure...' to '3 infrastructure...'

line 45 above - replace 'has' with 'as' in the sentence "...Linux host has a load balancer..."

File: playbooks/installation/installation.adoc [hunk 2]
```diff
@@ -50,28 +50,32 @@ The Smart Start architecture requires the following Servers(VMs) be provisioned:
...
   ** 2 cores
   ** 4 GB Memory
   ** 10 GB Root Drive
 
+TODO: Need something about persistent storage
+- push for iSCSI volumes for registry/metrics/logging
+
 NOTE: We chose the resource sizing above based on some reasonable starting sizes for hosts based on our experiences in the field. Before building further environments, it is important to go through a real production sizing exercise. For more information on how we recommend going about this, see our link:/playbooks/operationalizing/environment_sizing{outfilesuffix}[Environment Sizing Guide].
 
 NOTE: We recommend segregating local disks this way in order to provide more stability to the separate components that will write to these locations. For more information about why we configure local storage this way, read link:/playbooks/operationalizing/local_storage{outfilesuffix}[Considerations for Configuring Local Storage on OpenShift Hosts]
```
Comment (JaredBurck) [inline playbooks/installation/installation.adoc]: line 100  below - replace 'two' with 'three' in the 4th bullet point for the infrastructure nodes

File: playbooks/installation/installation.adoc [hunk 3]
```diff
@@ -123,16 +127,16 @@ Overall requirements for Installing OpenShift are very simple:
 * Ensure time is in sync on all nodes
```
Comment (JaredBurck) [inline playbooks/installation/installation.adoc]: Are lines 118 and 119 above still relevant or should this be removed?
```
* If you plan to Install HA Masters, an additional channel is needed on Master Nodes
  * rhel-7-for-ha-server-rpms
```

**Comment by sabre1041** [comment]
@JaredBurck How do you want to proceed?

**Comment by etsauer** [comment]
@JaredBurck made changes you requested.

**Comment by JaredBurck** [comment]
merged!

**Pull request merged**